sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1530,10 +1531,19 @@ def _update_cluster_region(self):
             self.launched_resources = self.launched_resources.copy(
                 region=region)
 
+        @property
+        def cluster_yaml(self):
+            return os.path.expanduser(self._cluster_yaml)
+
         def __setstate__(self, state):
+            self._version = self._VERSION
+
             version = state.pop('_version', None)
```

Comment:
nit: for readability, can we rename this to `old_version = state.pop('_version', None)`. Because new version has been set to `self._VERSION` in the above line